packages/lesswrong/components/votes/EAReactsSection.tsx

```diff
@@ -199,19 +204,38 @@ export const isEAReactableDocument = (
   return collection === Posts || collection === Comments;
 }
 
-const EAReactsSection: FC<{
+type EAReactsSectionOptions = {
+  // viewOnly disables all interactivity, including tooltips
```

Comment:
I wanted to show the reacts on the user's top comment without making anything interact-able. Let me know if this is confusing or you spot any issues here.

Reply:
Disabling tooltips was mostly to get this done more quickly, plus we expect mostly mobile usage and tooltips are slightly buggy there (maybe no longer). Since we are showing the user their own comment / quick take I don't want them voting on it anyway, and don't want them accidentally clicking to vote on it on mobile.